### DISCUSSION TITLE
Homepage - Custom.js not working

### DISCUSSION THREAD

**Discussion by ejohnnyro** [tteck/Proxmox#3950] [Help Needed]
I have been trying to add some custom functionality to homepage. I want to read some things from proxmox API and then display them in homepage. Sadly custom.js doesn't seem to work. I tried a simple console log command and it's not being executed. 

I am using the latest version 0.9.11.

I tried to figure out why on my own but after a full day of reading, research and testing. I can't understand why the custom.js is empty. Tho i have manually update it in /config/custom.js

Help?

**Comment by Scorpoon** [comment]
Seems more related to [homepage](https://github.com/gethomepage/homepage) instead of tteck's proxmox scripts.
Homepage already has a widget for proxmox, maybe it is what you are looking for: [https://gethomepage.dev/widgets/services/proxmox/](https://gethomepage.dev/widgets/services/proxmox/)

**Reply by ejohnnyro** [reply]
Oh I am sorry. I used the script from https://tteck.github.io/Proxmox/#dashboards and for some reason it took me here when i Googled. The default Proxmox widget works but it doesn't bring the data i need. It's very limited.
